2016/proposals/maurya-kumar-raj.md

```diff
@@ -0,0 +1,71 @@
+# Add taxonomic name resolution to the EcoData Retriever to facilitate data science approaches to ecology
+
+## Abstract
+
+The EcoData Retriever is an automated software that is used for finding, downloading, and cleaning up ecological data files, and then organizes the whole data in a local database of your choice. The program cleans and restructures datasets in normalized form and formats them according to the suitable standards of the database management systems.
+
+This project deals with handling the problem of resolving scientific names that are stored in appropriate manner across various datasets. This software would enable its users to conveniently combine different datasets that will be helpful in answering scientific queries. 
+
+## Technical Details
+
+EcoData Retriever is python based tool for downloading, cleaning up, and restructuring ecological data. The information about the repositories are stored in .script files which contain URLs to the files containing the datasets.
+
+This project would build a system for sending a species name to one or more of the taxonomic name resolution services and determine the best name to appear in the generated databases. The project is composed of the following main components: 
```

Comment:
This project would include a system for sending a species name to one or more of the taxonomic name resolution services and determine the best name to appear in the generated databases.